rerun_py/rerun_sdk/rerun/components/tensor_data.py

```diff
@@ -12,8 +12,6 @@
 
 
 class TensorData(datatypes.TensorData):
-    # You can define your own __init__ function as a member of TensorDataExt in tensor_data_ext.py
```

Comment:
Nit: this comment should still be here.  There may be a valid reason to implement an Extension for a delegating component (even if we don't yet).